packages/noya-utils/src/__tests__/sortBy.test.ts

```diff
@@ -3,3 +3,83 @@ import { sortBy } from '../index';
 test('empty array', () => {
   expect(sortBy([], '')).toEqual([]);
 });
+
+test('first key', () => {
+  expect(
+    sortBy(
+      [
+        { name: 'Liam', last: 'Smith' },
+        { name: 'Noah', last: 'Benjamin' },
+        { name: 'Sophia', last: 'Brown' },
+        { name: 'Emma', last: 'Jones' },
+        { name: 'Ava', last: 'Davis' },
+      ],
+      'name',
+    ),
+  ).toEqual([
+    { name: 'Ava', last: 'Davis' },
+    { name: 'Emma', last: 'Jones' },
+    { name: 'Liam', last: 'Smith' },
+    { name: 'Noah', last: 'Benjamin' },
+    { name: 'Sophia', last: 'Brown' },
+  ]);
+});
+
+test('second key', () => {
+  expect(
+    sortBy(
+      [
+        { name: 'Liam', last: 'Smith' },
+        { name: 'Noah', last: 'Benjamin' },
+        { name: 'Sophia', last: 'Brown' },
+        { name: 'Emma', last: 'Jones' },
+        { name: 'Ava', last: 'Davis' },
+      ],
+      'last',
+    ),
+  ).toEqual([
+    { name: 'Noah', last: 'Benjamin' },
+    { name: 'Sophia', last: 'Brown' },
+    { name: 'Ava', last: 'Davis' },
+    { name: 'Emma', last: 'Jones' },
+    { name: 'Liam', last: 'Smith' },
+  ]);
+});
+
+test('same value', () => {
+  expect(
+    sortBy(
+      [
+        { name: 'Liam', last: 'Smith' },
```

Comment:
At this point we've already tested different keys, so we can probably remove `last` to keep it simpler

Reply:
Actually I guess we wouldn't be able to tell them apart without `last`...

Reply:
so I keep them both? or only the first one?

Reply:
Let's keep it the way it is now